src/common/mixins/RealtimeLayerMixin.ts

```diff
@@ -86,7 +89,7 @@ export type RealtimeLayerMixinOptions = OlLayerOptions & {
 /**
  * RealtimeLayerInterface.
  */
-export class RealtimeLayerInterface {
```

Comment:
The new name sounds a bit generic to me ;)

Reply:
Opps, no idea how that happened